### ISSUE TITLE
Error Generating Metadata for existing images

### ISSUE THREAD

**Issue by CryptoPique** [HashLips/hashlips_art_engine#596]
When i run the function: npm run Generate_metadata, all the .json file are created. The problem is that when i upload to the candy machine and i mint the NFT i didn't get any trait, even if i've created them, the json file looks like this:

{
  "name": "The Solenoids #1",
  "symbol": "SED",
  "description": "The Solenoids were born, a whole new existence of 777 solenoids landed on the Solana Blockchain",
  "image": "1.png",
  "edition": 1,
  "attributes": [
    [
      {
        "trait_type": "average color",
        "value": "rgb(170,140,91)"
      },
      {
        "trait_type": "Who am i?",
        "value": "Im a Solanoid"
      },
      {
        "trait_type": "Rank",
        "value": 543
      }
    ]
  ],
  "properties": {
    "creators": [
      {
        "address": "X",
        "share": 100
      }
    ]
  },
  "compiler": "Hashlip art Engine"
}

I've made a check on a NFT showing traits and i've seen that the .json file have 1 single [ and not [ [ at the start of the attributes, so the problem should be that the Generate_metadata is returning a double [.
Any one have any idea how to change this ?

**Comment by SuperGeniusSidhant10** [comment]
I think that you have ran regenerate metadata you might want to update the image url but it is having some problem and it is updated by update_info.js in utils folder

**Comment by CryptoPique** [comment]
No, the "npm run generate_metadata" create new .json file for the image in your folder and it get the functions from generate_metadata.js

**Comment by baringji** [comment]
I have this script that preserve the existing metadata. However it will reorder your assets. Just put this in the utils folder and just run it.

https://gist.github.com/baringji/5a5a0fc90a6629e67047b1d5c029e923
